playbooks/ansible.cfg

```diff
@@ -7,4 +7,8 @@
 jinja2_extensions=jinja2.ext.do
 host_key_checking = False
 roles_path=../../ansible-roles/roles:../../ansible-private/roles:../../ansible-roles/
-ansible_managed=This file is created and updated by ansible, edit at your peril
\ No newline at end of file
+hostfile = ./ec2.py
+log_path = .ansible.log
+private_key_file = ~/.ssh/id_rsa
+remote_user = ubuntu
+ansible_managed=This file is created and updated by ansible, edit at your peril
```

Comment:
TODO: We can leave this unchanged. Separate PR will add this explicitly for each env.